client/app/admin/admin.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="container-fluid fill-height p-0">
+
+  <div class="row half">
+    <div id="menuBar" class="col-3 sidebar pl-0">
+      <button class="savebtn" (click)="saveMenu()">Save</button>
+      <button class="closebtn" (click)="closeMenuNav()">Edit Menu -</button>
+      <div class="sideScroll">
+        <table class="table table-hover table-sm" width="100%">
+          <thead>
+            <tr>
+              <th>Item</th>
+              <th>Price</th>
+              <th>Category</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let item of menu">
+              <td contenteditable="true">{{ item.name }}</td>
+              <td contenteditable="true">£{{ item.price }}</td>
```

Comment:
@jessicawong217 @last1here @Nadiajayy @chloehodgson93 we need to keep the name of the price consistent I think. Currently is price per portion in the orders data. Here is just price. I'm happy with either one as long as is consistent everywhere.

Reply:
I think price_per_portion is good as it should be as specific as possible regarding units within db.